specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/examples/BillingAccountAlerts.json

```diff
@@ -0,0 +1,98 @@
+{
+  "parameters": {
+    "api-version": "2018-08-01-preview",
+    "billingAccountId": "12345:6789",
+    "scope": "providers/Microsoft.Billing/billingAccounts/12345:6789"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "providers/Microsoft.Billing/billingAccounts/12345:6789/providers/Microsoft.CostManagement/alerts/00000000-0000-0000-0000-000000000000",
```

Comment:
>providers/Microsoft.Billing/billingAccounts/12345:6789/providers/Microsoft.CostManagement/alerts/00000000-0000-0000-0000-000000000000 [](start = 19, length = 133)

What happens when the user does a GET on the resource ID?

Reply:
they receive the correct object response

Reply:
Unless I'm missing something, the swagger does not declare a path for .../alerts/{name}

---
In reply to: [419817404](https://github.com/Azure/azure-rest-api-specs/pull/9263#discussion_r419817404) [](ancestors = 419817404)

Reply:
Does your service implement this path and it's just missing from swagger.

---
In reply to: [419824483](https://github.com/Azure/azure-rest-api-specs/pull/9263#discussion_r419824483) [](ancestors = 419824483,419817404)

Reply:
Discussed over IM: The service implements this but it's missing from the swagger.

---
In reply to: [419824926](https://github.com/Azure/azure-rest-api-specs/pull/9263#discussion_r419824926) [](ancestors = 419824926,419824483,419817404)